iati-activities-schema.xsd

```diff
@@ -2519,17 +2506,19 @@
     </xsd:complexContent>
   </xsd:complexType>
 
-<!-- resultLocationBase extends locationBase to include the common documentation for location as a sub-element of the iati-activity level result element -->
+<!-- resultLocationBase provides the common ref attribute for all location elements in the iati-activity/results element. -->
   <xsd:complexType name="resultLocationBase">
-    <xsd:complexContent>
-      <xsd:extension base="locationBase">
-        <xsd:annotation>
-          <xsd:documentation xml:lang="en">
-            A location already defined and described in the iati-activity/location element.
```

Comment:
This definition appears to have been removed, an now longer exists, unless I've missed something...?

Reply:
Fixed in 9e86150